### PR TITLE
NF: remove "new Object[]" in queries

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
For some reason, some new Object[] remained. There was actually a really simple way to remove them and clean the code

### CODE REVIEW

**Comment by Arthur-Milchior** [comment]
> Not that it makes the actual PR better or worse but I am now more comfortable personally as a programmer with varargs (java) and spread operators (javascript) and such, so I intuitively understand this language feature now.
> 
> And it LGTM

FYI: Seems quite similar to https://github.com/ankidroid/Anki-Android/pull/6709#issuecomment-662106949

File: AnkiDroid/src/main/java/com/ichi2/libanki/Note.java [hunk 1]
```diff
@@ -156,8 +156,7 @@ public int numberOfCards() {
     }
 
     public List<Long> cids() {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Note.java:158]: Line 154 as well

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Note.java:158]: Done

**Comment by Arthur-Milchior** [comment]
I've checked for all occurrence of "new Object[]" and tried to find the one which seems related to a query. Some of them are related to queries but not in DB so I can't remove them. Some other are related to TaskData; a lot of them. Hopefully, they'll disappear in some time. But I don't expect it to occur immediately. I don't know how to check and fix this only

**Comment by david-allison-1** [comment]
Best way to search would be a find usages on the methods, or an Android studio coffee analysis on the refactoring over the entire project

**Comment by Arthur-Milchior** [comment]
Great. That was my idea too. But that would be nice that I discover tools David mention too :)

**Pull request merged**